charts/rpe-send-letter-service/Chart.yaml

```diff
@@ -1,4 +1,5 @@
 name: rpe-send-letter-service
+apiVersion: v1
```

Comment:
any context?

Reply:
Please check this link https://hmcts-reform.slack.com/archives/C6DN09J8G/p1571820031313300